.github/workflows/smoketest.yml

```diff
@@ -449,3 +450,84 @@ jobs:
         sudo update-locale
         tar --zstd -xf install.tgz
         install/opt/HPCCSystems/bin/unittests
+
+  # NB: this doesn't really need the post-setup data files included in install.tgz
+  # but as this is relatively quick and in parallel with others, it probably doesn't matter
+  eclwatchuitests:
```

Comment:
trivial: for consistency/readability can you make this eclwatch-ui-tests

Reply:
Changed